### PR TITLE
W-16047302-Add-default-debugger-settings-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by juarez-fabian — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/visual-debugger-concept.adoc [hunk 1]
```diff
@@ -3,11 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio debugger enables you to run your application from Studio in debug mode, and to configure breakpoints to stop the execution of your application at a specific message processor. Then, using the *Mule Debugger* view, you can inspect the content of the message as it exists at that point in the flow, and evaluate a DataWeave expression against it.
+With Anypoint Studio debugger, you can run your application from Studio in debug mode and configure breakpoints to stop the execution of your application at a specific message processor. In the *Mule Debugger* view, inspect the content of the message as it exists at that point in the flow and evaluate a DataWeave expression against it.
```
Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc]:
```suggestion
With Anypoint Studio debugger, you can run your application from Studio in debug mode and configure breakpoints to stop application execution at a specific message processor. In the *Mule Debugger* view, inspect the content of the message as it exists at that point in the flow and evaluate a DataWeave expression against it.
```

File: modules/ROOT/pages/visual-debugger-concept.adoc [hunk 2]
```diff
@@ -3,11 +3,11 @@ ifndef::env-site,env-github[]
...
 
-The debugger works with Enterprise Edition runtimes and does not connect to Community Edition runtimes.
+The debugger works with Enterprise Edition runtimes and doesn't connect to Mule Kernel runtimes.
```
Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc:8]: Kernel init cap correct?

File: modules/ROOT/pages/visual-debugger-concept.adoc [hunk 3]
```diff
@@ -17,18 +17,38 @@ When you run your application in debug mode for the first time, Anypoint Studio
...
-. Select *Debug As* > *Mule Application*. +
-Studio asks you for permission to switch to the *Mule debug* perspective.
+. Right-click your project in *Package Explorer*.
+. Select *Debug As* > *Mule Application*.
++
+Studio requests permission to switch to the *Mule debug* perspective.
 . Click *Yes*.
 
-Additionally, you can run your application in debug mode by right-clicking in the canvas and selecting *Debug Project*, or by clicking the *Debug* button on your top toolbar image:to-run-debug-mode-e7b7b.png[debug-button].
+Additionally, run your application in debug mode by right-clicking in the canvas and selecting *Debug Project*, or by clicking the *Debug* button on the toolbar image:to-run-debug-mode-e7b7b.png[debug-button].
```
Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc]: Do you mean "Alternatively"?

Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc]:
```suggestion
Alternatively, run your application in debug mode by right-clicking in the canvas and selecting *Debug Project*, or by clicking image:to-run-debug-mode-e7b7b.png[""] (*Debug*) on the toolbar.
```

File: modules/ROOT/pages/visual-debugger-concept.adoc [hunk 4]
```diff
@@ -17,18 +17,38 @@ When you run your application in debug mode for the first time, Anypoint Studio
...
 
-When debugging in Studio, the debugger listens for incoming TCP connections on `localhost` and port `6666`. You can modify this configuration in by clicking *Run* > *Debug configurations* and selecting the *Debug* tab.
+Enable precise control over application debugging by setting the debugger connections and maximum values. To modify the default configuration values, click *Run* > *Debug configurations* and select the *Debug* tab.
+
+[%header,%autowidth.spread,cols="a,a,a"]
+|===
+| Option | Description | Default Value
+
+| Connection Properties 
+| Studio debugger listens to incoming TCP connections on `localhost` and port `6666`. Pause the debugger listening activities by selecting *Suspend* to prevent the debugger from triggering or processing any incoming requests temporarily.
```
Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc]:
```suggestion
| Studio debugger listens to incoming TCP connections on `localhost` and port `6666`. To temporarily prevent the debugger from triggering or processing incoming requests, click *Suspend* to pause debugger listening activities. 
```

Comment (hannanelson) [reply modules/ROOT/pages/visual-debugger-concept.adoc]: Do you need the font changes here?

File: modules/ROOT/pages/visual-debugger-concept.adoc [hunk 5]
```diff
@@ -17,18 +17,38 @@ When you run your application in debug mode for the first time, Anypoint Studio
...
+[%header,%autowidth.spread,cols="a,a,a"]
+|===
+| Option | Description | Default Value
+
+| Connection Properties 
+| Studio debugger listens to incoming TCP connections on `localhost` and port `6666`. Pause the debugger listening activities by selecting *Suspend* to prevent the debugger from triggering or processing any incoming requests temporarily. 
+| 
+
+* TCP connections: `localhost`
+* Port: `6666`
```
Comment (hannanelson) [inline modules/ROOT/pages/visual-debugger-concept.adoc:41]: These bullets duplicate the first sentence in the description. Any way to address and make parallel with the other descriptions (imperative)?

File: modules/ROOT/pages/visual-debugger-concept.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc]: OK! Thanks!

File: modules/ROOT/pages/visual-debugger-concept.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc:8]: According to [this entry](https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#:~:text=or%20%22Mule%20runtimes%22-,Mule%20Kernel,-Replaces%20Mule%20Community) in the [Term Presentation Reference](https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference), yes.

File: modules/ROOT/pages/visual-debugger-concept.adoc (hunk 3 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc]: You're right!

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc]: OK! Thanks!

File: modules/ROOT/pages/visual-debugger-concept.adoc (hunk 4 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc]: So sorry, but I think I don't understand your comment :/

File: modules/ROOT/pages/visual-debugger-concept.adoc (hunk 5 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/visual-debugger-concept.adoc:41]: Let's see if my wording works.

**Review by kevintroller — APPROVED** [review]

**Pull request merged**